### PR TITLE
Put a default title on main landing page and remove invalid line breaks

### PR DESCRIPTION
**Pull request by benjaoming**

### CODE REVIEW

File: kalite/distributed/templates/distributed/base.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <meta name="wot-verification" content="075abe861486e6bb3ab9"/>
         <meta name="viewport" content="width=device-width, initial-scale=1">
         {% block titletag %}
-        <title>{% block title %}{% trans title %}{% endblock title %} | KA Lite</title>
+        <title>{% block title %}{% trans "Welcome" %}{% trans title %}{% endblock title %} | KA Lite</title>
```
Comment (MCGallaspy) [inline kalite/distributed/templates/distributed/base.html]: Why did you add the Welcome here? Does that create a grammatically correct title?

Comment (benjaoming) [inline kalite/distributed/templates/distributed/base.html]: It's a default so that pages without a title have something rather than " | KA Lite"

Comment (jamalex) [inline kalite/distributed/templates/distributed/base.html]: IT'S @BENJAOMING! HE SURVIVED HIS TRIP!!!!!

**Comment by MCGallaspy** [comment]
Oh I see, it's overridden by templates that inherit from this.

File: kalite/distributed/templates/distributed/base.html (hunk 1 above)
Comment (benjaoming) [inline kalite/distributed/templates/distributed/base.html]: No no, this is the us government, we just maintain his online identity while he's gone... it's a new service :)

Comment (aronasorman) [inline kalite/distributed/templates/distributed/base.html]: Cool, is this new service webscale?
